packs/_source/races/gnome/rock-gnome.json

```
@@ -90,6 +90,23 @@
         },
         "level": 0,
         "title": "",
+        "icon": null,
+        "value": {}
+      },
+      {
+        "_id": "3iZH8gdahsVwLPbQ",
+        "type": "Trait",
+        "configuration": {
+          "mode": "default",
+          "allowReplacements": false,
+          "grants": [
+            "tool:art:tinker"
+          ],
+          "choices": [],
+          "hint": ""
+        },
+        "level": 0,
+        "title": "",
```

We should give this a name to match the feature, same way the description lists it.

```suggestion
        "title": "Tinker",
```